### PR TITLE
.gitignore + kind + cert-manager v1.0.0

### PR DESCRIPTION
**Pull request by mythi**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -44,8 +44,9 @@ test-with-kind:
 	@$(PODMAN) tag localhost/intel/intel-fpga-admissionwebhook:devel docker.io/intel/intel-fpga-admissionwebhook:devel
 	@mkdir -p $(e2e_tmp_dir)
 	@$(PODMAN) save "docker.io/intel/intel-fpga-admissionwebhook:devel" -o $(e2e_tmp_dir)/$(WEBHOOK_IMAGE_FILE)
-	@$(KIND) create cluster --name "intel-device-plugins" --kubeconfig $(e2e_tmp_dir)/kubeconfig --image "kindest/node:v1.17.0"
+	@$(KIND) create cluster --name "intel-device-plugins" --kubeconfig $(e2e_tmp_dir)/kubeconfig --image "kindest/node:v1.19.0"
 	@$(KIND) load image-archive --name "intel-device-plugins" $(e2e_tmp_dir)/$(WEBHOOK_IMAGE_FILE)
+	kubectl --kubeconfig=$(e2e_tmp_dir)/kubeconfig apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.15.0/cert-manager.yaml
```
Comment (rojkov) [inline Makefile:49]: In some envs kubectl may be absent in `PATH`. Better to use `$(KUBECTL)`.

**Review by bart0sh — APPROVED** [review]

**Review by rojkov — APPROVED** [review]

**Comment by mythi** [comment]
@kad needs your approval

**Review by kad — APPROVED** [review]
lgtm

**Comment by mythi** [comment]
hold: I'll move to cert-manager v1.0.0

**Review by rojkov — APPROVED** [review]

**Pull request merged**